templates/content/field--oe-document-reference--oe-document--oe-document.html.twig

```diff
@@ -0,0 +1,13 @@
+{#
+/**
+ * @file
+ * Field template of the document reference entity.
```

Comment:
I already commented on this before but it got lost when the file was renamed.
Well, not _lost_ really, it was still visible in the "Conversation" tab - please always check that!
Not sure how exactly, discussion in chat ongoing.